ui/components/src/BarChart/BarChart.tsx

```diff
@@ -31,11 +36,12 @@ export interface BarChartProps {
   width: number;
   height: number;
   data: BarChartData[] | null;
-  unit: UnitOptions;
+  unit?: UnitOptions;
+  mode?: ModeOption;
 }
 
 export function BarChart(props: BarChartProps) {
-  const { width, height, data, unit } = props;
+  const { width, height, data, unit = { kind: 'Decimal' }, mode = 'value' } = props;
   const chartsTheme = useChartsTheme();
 
   const option: EChartsCoreOption = useMemo(() => {
```

Comment:
Seeing some cutoff with longer axis labels, adjusting grid.left or one of the axisLabel options might help

<img width="596" alt="image" src="https://github.com/perses/perses/assets/9369625/455efcea-f961-41a1-ade0-4e01f246c3b2">

Reply:
I think we should adjust default to show 'M', 'B' notation on value axis like TimeSeriesChart 

<img width="630" alt="image" src="https://github.com/perses/perses/assets/9369625/e31b194f-2ec9-4ca6-8772-de9ef54d3668">

Reply:
Noticed some cutoff on right side when testing percent mode, I think those [grid](https://echarts.apache.org/en/option.html#grid) properties will help, surprised the theme defaults don't cover us though 😕 

<img width="962" alt="image" src="https://github.com/perses/perses/assets/9369625/cda57bf0-4075-4333-9668-99cb4a2aedf6">